build.sbt

```diff
@@ -1289,7 +1289,7 @@ lazy val soccoSettings = if (sys.env.contains("SOCCO")) {
       "-P:socco:package_com.spotify.scio:https://spotify.github.io/scio/api"
     ),
     autoCompilerPlugins := true,
-    addCompilerPlugin("com.criteo.socco" %% "socco-plugin" % "0.1.9"),
+    addCompilerPlugin(("io.regadas" %% "socco-ng" % "0.1.1").cross(CrossVersion.full)),
```

Comment:
Did you have to fork it ? Maybe we should consider migrating to another lib if socco is not maintained.

Reply:
Yeah I did. The original one seems defunct as nobody responds to PRs. Having a different tool is a possibility, although the idea of having the code become document grew on me, and that’s why I forked it to keep it a bit alive and updated. I actually have some extra ideas to add to it.